new-log-viewer/package.json

```diff
@@ -5,10 +5,17 @@
   "main": "src/index.tsx",
   "scripts": {
     "build": "webpack --config webpack.prod.js",
+    "start": "webpack serve --open --config webpack.dev.js",
+
     "lint": "npm run lint:check",
-    "lint:check": "eslint src webpack.*.js --max-warnings 0",
-    "lint:fix": "npm run lint:check -- --fix",
-    "start": "webpack serve --open --config webpack.dev.js"
+    "lint:check": "npm-run-all --sequential --continue-on-error lint:css-check lint:js-check",
+    "lint:fix": "npm-run-all --sequential --continue-on-error lint:css-fix lint:js-fix",
+
+    "lint:css-check": "stylelint src/**/*.css",
+    "lint:css-ci": "npm run lint:css-check -- --formatter github",
```

Comment:
Although the GitHub formatter has been deprecated according to the Stylelint docs [1], the default `string` formatter does not work with GitHub action problem matchers because the formatter outputs relative paths instead of absolute paths of the source files. There has been calls for behaviour change about resolving the paths of problem matcher since its debut, but the proposals have yet to be implemented [2]. Before we move the new-log-viewer files out to the project root, using the `github` formatter might be the quickest way.

[1] https://stylelint.io/user-guide/options#formatter
[2] https://github.com/actions/runner/issues/765

Reply:
https://www.npmjs.com/package/stylelint-actions-formatters#:~:text=This%20package%20should%20be%20used%20with%20the%20stylelint%2Dproblem%2Dmatcher%20action.

> This package should be used with the [stylelint-problem-matcher](https://github.com/xt0rted/stylelint-problem-matcher) action.

It seems that to get the whole flow working, we also need to use a custom GitHub action in the workflow, which can be less ideal.

Do you think we can keep using the `github` formatter for now? I have already had a Stylelint problem matcher drafted which works perfectly when we run Stylelint with the default `string` formatter from the project root. Hopefully in the coming days, when we move the files from `/new-log-viewer` into the project root, we can replace this `github` formatter with the `string` formatter and start using the problem matcher. What do you think?